cmd/config.go

```diff
@@ -75,9 +75,12 @@ type Config struct {
 
 func (c Config) Apply(cfg Config) Config {
 	c.Options = c.Options.Apply(cfg.Options)
-	if cfg.Out.Valid {
-		c.Out = cfg.Out
+	for _, o := range cfg.Out {
+		if o.Valid {
+			c.Out = append(c.Out, o)
```

Comment:
This would make it so there's no way to actually override a configured collector from a lower level. If we (after fixing https://github.com/loadimpact/k6/issues/587) specify a collector in the exported script options, there wouldn't be a way to actually override it from the CLI options, like we could do with all the other options.